evm-contracts/src/v0.6/dev/AccessControlledAggregator.sol

```diff
@@ -105,10 +105,10 @@ contract WhitelistedAggregator is FluxAggregator, Whitelisted {
    * maxSubmissions) answer and updatedAt may change between queries.
    */
   function getRoundData(uint256 _roundId)
-    external
+    public
```

Comment:
For my own edification, why this change from `external` to `public`?

Reply:
Solidity made it easier to use functions marked as public in the implementation and external in the interface. This allowed us to get rid of the `_getRoundData(...)` internal functions which would be called by inheriting contracts and instead we can simply use `super.getRoundData(...)`.